lib/sequelize.js

```diff
@@ -1093,5 +1119,7 @@ module.exports = (function() {
     this.connectionManager.close();
   };
 
+  // Allows the promise to access cls namespaces
+  Promise.Sequelize = Sequelize;
```

Comment:
Is the cls "leak" on multiple sequelize instances issue fixed?
On @findhit we have a process with two sequelize instances.